tutorials/tutorial_1/containers/prediction-model/app/config/config_fed_client.json

```
@@ -1,22 +1,20 @@
 {
   "format_version": 2,
-
   "executors": [
     {
-      "tasks": ["train", "submit_model"],
+      "tasks": [
+        "train"
+      ],
       "executor": {
-        "path": "pneumonia_trainer.PneumoniaTrainer",
+        "path": "nvflare.app_opt.pt.in_process_client_api_executor.PTInProcessClientAPIExecutor",
         "args": {
-          "lr": 0.01,
-          "epochs": 1
+          "task_script_path": "/home/localuser/app/custom/pneumonia_fl_client_api.py",
+          "server_expected_format": "pytorch"
         }
       }
     }
   ],
-  "task_result_filters": [
-  ],
-  "task_data_filters": [
-  ],
-  "components": [
-  ]
-}
+  "components": [],
+  "task_data_filters": [],
+  "task_result_filters": []
+}
```

Nit: End-of-line at the end (in some other files as well)
```suggestion
}

```

---

ok good to know